core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -380,20 +381,36 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       case err: JsValue  => AttributeString(err.toString)
     }
 
-  private def resolveDrsUriServiceAccounts(drsUris: Set[String], userInfo: UserInfo)(implicit
+  def resolveDrsSignedUrls(drsUris: Set[String], userInfo: UserInfo)(implicit
     executionContext: ExecutionContext
-  ): Future[Set[String]] =
+  ): Future[Set[String]] = {
+
+    val urisByProvider: Map[Option[String], List[String]] = drsUris.toList.groupBy(DrsResolver.getProvider)
+
+    if (urisByProvider.contains(None)) {
+      logger.warn(s"Some URIs were unparsable: ${urisByProvider(None)}") // TODO What should be logged vs. thrown?
+      throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.NotFound, "Unable to parse URI"))
```

Comment:
This is a user input issue. I don't think we should log anything but if we do, it should be at either info or debug level. I think the status code should be BadRequest, NotFound is generally that the url of the api request does not exist. I would include the unparsable uris in the error message, or maybe the first 5 or something.

Alternatively, we ignore these, don't validate them, and let something downstream fail if it is going to.